Source/WebCore/workers/service/context/ServiceWorkerThread.cpp

```diff
@@ -116,8 +116,14 @@ static WorkerParameters generateWorkerParameters(const ServiceWorkerContextData&
     };
 }
 
+#if ENABLE_REMOTE_INSPECTOR
+static const WorkerThreadStartMode serviceWorkerThreadStartMode = WorkerThreadStartMode::WaitForInspector;
+#else
+static const WorkerThreadStartMode serviceWorkerThreadStartMode = WorkerThreadStartMode::Normal;
```

Comment:
ideally we'd have this only happen if `DeveloperExtrasEnabled` as most users dont even have that turned on and therefore shouldn't have to "suffer" a slower startup

NIT: these could be `constexpr`